### DISCUSSION TITLE
可以对多列进行预测嘛？

### DISCUSSION THREAD

**Discussion by xxm1668** [ourownstory/neural_prophet#1540] [Feature Requests & Feedback]

**Comment by ourownstory** [comment]
@xxm1668 
If I understand right, you are asking: "Can I predict multiple columns?"

You can regress (input) multiple time series, but you can only predict one series at a time (target).
You can however, build a shared model (global modeling) across multiple series.

**Comment by xxm1668** [comment]
thanks
